finance/proposal-calls/cycle5/cycle5.md

```diff
@@ -0,0 +1,108 @@
+
+# Funding goals
+Funding requests (FR) should address items on the Astropy Project’s Roadmap. In particular, funding will be prioritized for “red” and “orange” roadmap items, which have been identified by the community as being high priority.
+
+FRs shall foster the Astropy Project goals of community, shared work, and cooperation and shall be described, reviewed, and executed in this manner.
+
+# Funding amount
+
+We anticipate a maximum of **$XXX,XXX** (US) to be available for this funding cycle. 
+
+# Period of Performance
+
+The period of performance should be for one year. Ideally, Jan 1–Dec 31, 2026.
+
+# What can the funding be used for?
+
+Money can be used for travel, subscriptions, etc, in addition to direct financial support to individuals, including individuals based outside of the United States.
+
+Funding requests may be submitted even if they include the recruitment and hiring of another person to complete the task. See below: Open Hire Process.
+
+The funds currently come from a NASA ROSES grant. NumFocus, the Project’s fiscal sponsor, and the Finance Committee assume responsibility for ensuring that all budgets are consistent with any applicable funding agency rules.
+
+# Funding Request Process Overview
+
+We envision a two stage FR process to enable the community to discuss and improve ideas, to merge teams with similar ideas, and for the SPOC to ensure as much of the Roadmap as possible is being addressed.
+
+* Oct 17, 2025: **_Draft_** FR Deadline 
+* Three week iteration and discussion period during which draft FRs can be fleshed out and modified. During this time:
+  * The SPOC may submit and/or solicit new proposals that fill gaps not addressed by the initial round of requests. 
+  * The Finance Committee will review budgets and may provide feedback and/or request more information. 
+* Nov 7, 2025: Finalized FRs are due, two week Voting period begins
+* Nov 24, 2025: The SPOC and Finance Committee will work together to select funding requests and allocate budgets. 
+* Dec 8, 2025: Announcement of selected FRs. Drafting contracts begins.
+* Jan 1, 2026: Work commences
+
+Note that these dates are nominal, and unplanned events like a government shutdown could extend some of the dates/deadlines.
+
+We aim to achieve broad consensus before the vote. Requesters are encouraged to combine ideas and teams, modify, or withdraw FRs during the discussion period, as appropriate. Discussion is open to anyone; it is not restricted to voting members. 
+
+Voting members of the Astropy Project vote on the FRs using thumbs up/down on a dedicated comment. Every voting member of the Astropy Project has equal voting rights.
```

Comment:
I think I asked this in the past, but are we going to actually check the votes against the voting member list? If so maybe could then also say "Non-voting members are free to vote on FRs, but these votes will not be included in the count to determine funding allocations"